src/audio/dai-zephyr.c

```diff
@@ -547,7 +547,8 @@ static int dai_playback_params(struct comp_dev *dev, uint32_t period_bytes,
 	dd->z_config = dma_cfg;
 
 free:
-	dma_sg_free(&config->elem_array);
```

Comment:
The fixes tag contains wrong sha, it should be:
```
Fixes: 3d0e327b4bed ("dai-zephyr: free allocated memory in error paths")
```

Reply:
fixed now